### PR TITLE
Forward host ports to the db and es services

### PR DESCRIPTION
**Pull request by nickdaugherty**
To access ES directly from the host, and to have a static MariaDB host port, add the `portforward` configs to `.lando.yml`.

## Test Steps

1. `lando rebuild --yes`
1. `lando info` - should show both the `vip-search` and `database` services as having `19200` and `13306` exposed in their `external_connection` info.

### CODE REVIEW

File: .lando.yml [hunk 1]
```diff
@@ -42,6 +42,7 @@ services:
     type: elasticsearch:custom
     overrides:
       image: bitnami/elasticsearch:7.8.0
+    portforward: 13306
```
Comment (netsuso) [inline .lando.yml:45]: Hehehe, you set the same port as MySql 🙃

```suggestion
    portforward: 19200
```

Comment (nickdaugherty) [reply .lando.yml:45]: Copy pasta!

**Comment by nickdaugherty** [comment]
Good point on the multi-instance stuff. Technically on VIP Go we run a shared ES service, so it might make sense to leave that one as standalone (not a new service per instance). The proxy stuff in Lando looks cool.

For MySQL, I like to use Sequel Pro during local dev, which is why I was hoping to have a fixed port. I wonder if we could increment ports for each subsequent instance? So that they are at least predictable 🤔

**Comment by netsuso** [comment]
Probably it will not be possible (or at least it will not be easy) to have a common ES service for all Lando instances. The incrementing/predictable ports would be feasible, although I'm not sure if its complexity will be worth. Just in case, don't get too used to having a fixed port :sweat_smile: 

If you are not using any tool for ES, I suggest you to try that Head extension I linked above. It's much better than running curls and parsing jsons 😉

**Comment by nickdaugherty** [comment]
Nice I'll have to check out Head - the other one I've used in the past is Sense but looks like that has been folded into Kibana and isn't released as a browser extension now. Might be worth including Kibana to make it easier to see what's going on with the indexes 🤔

**Comment by mjangda** [comment]
For multi-tenant, I'm sure it's possible to create global shared services through Lando (similar to its proxy).

An alternate thought would be to to have the `vip-create` script pick unused ports and set those in the Lando config file.

**Comment by netsuso** [comment]
I prefer not to have a global shared service, as it will make it more difficult to understand and maintain for most users.

I think we can manage it in the creation script, indeed. At least once a port is assigned to a site during creation, it should be kept on any lando rebuild or destroy+start.

**Pull request merged**